presto-main-base/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -596,7 +596,8 @@ public static QueryStats create(
                 for (OperatorStats operatorStats : stageExecutionStats.getOperatorSummaries()) {
                     // NOTE: we need to literally check each operator type to tell if the source is from table input or shuffled input. A stage can have input from both types of source.
                     String operatorType = operatorStats.getOperatorType();
-                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) || operatorType.equals(MergeOperator.class.getSimpleName())) {
+                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) || operatorType.equals(MergeOperator.class.getSimpleName()) ||
+                    operatorType.equals(PrestoSparkRemoteSourceOperator.class.getSimpleName()) || operatorType.equals("ShuffleRead")) {
```

Comment:
I don't see a "ShuffleRead" operator in OSS, is this an internal Meta operator?

Reply:
> I don't see a "ShuffleRead" operator in OSS, is this an internal Meta operator?

It's a C++ operator: https://github.com/prestodb/presto/blob/9dfe7717d9f41b48cf6057ddfffb529990d3ad6d/presto-native-execution/presto_cpp/main/operators/ShuffleRead.h#L20